products/samsungmobile.md

```diff
@@ -137,10 +362,15 @@ releases:
     releaseDate: 2021-03-26
 
 -   releaseCycle: "Galaxy A72"
-    support: true
+    support: 2022-10-01 # Approximate from here: https://9to5google.com/2022/10/04/samsung-android-12-update-rollout/
```

Comment:
Upgradable to Android 12 according to https://www.gsmarena.com/samsung_galaxy_a72-10469.php. Should we consider it supported ?